src/tmdb.py

```
@@ -1025,18 +1020,24 @@ async def tmdb_other_meta(
-            logger.info(f"[bold red]Failed to fetch media data: {response.status_code}[/bold red]")
-            return {}
+        cache_key = json.dumps({"category": category, "id": tmdb_id}, sort_keys=True)
+        cached_media = await cache.get("tmdb", "main", cache_key)
+        if not is_cache_miss(cached_media) and isinstance(cached_media, dict):
+            media_data = cached_media
+        else:
+            response = await client.get(main_url, params={"api_key": tmdb_api_key})
+            try:
+                response.raise_for_status()
+                media_data = typing_cast(dict[str, Any], response.json())
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)